docs/source/configuration.rst

```diff
@@ -429,6 +432,15 @@ And here is an example of server-based authentication using SASL::
     auth_password = SopelIsGreat!  # your bot's password
     auth_target = PLAIN            # default sasl mechanism
 
+Example of authentication to a ZNC bouncer::
+
+    [core]
+    auth_method = server           # select server auth
+    # auth_username can't be used with server authentication, so instead
```

Comment:
Can't, or isn't? It isn't clear to me why.

I think making it more explicit by saying "Sopel doesn't use auth_username with `server` auth method." would make it clear as to **why** the user **cannot** use `auth_username`: because it **isn't** used by Sopel.

The rest of the sentence "so instead [...]" looks very good, no comment on that part.

Reply:
It _can't_, without inferring something like the colon-concatenation. The IRC protocol does not allow specifying an "authentication username", only your `USER`, `NICK`, and `PASS`, which are used for other things and configured elsewhere.

Reply:
Then I confirm: "can't" is a consequence of the rule, not the rule itself, which is: it *is not* used.

```suggestion
    # auth_username is not used with server authentication, so instead
```